etc/testing/circle/deploy-pachyderm.sh

```diff
@@ -9,5 +9,12 @@ helm repo add pachyderm https://pachyderm.github.io/helmchart
 helm repo update
 helm install pachd pachyderm/pachyderm --set deployTarget=LOCAL --version ${PACHYDERM_VERSION}
 
+# kubectl wait --for=condition=available deployment -l app=pachd --timeout=5m
+# kubectl wait statefulset.apps/pachd-loki --for=jsonpath='{.status.readyReplicas}'=1 --timeout=5m
+# There is a bug with wait for json path https://github.com/kubernetes/kubectl/issues/1236
+
+sleep 120s
 kubectl wait --for=condition=available deployment -l app=pachd --timeout=5m
+kubectl wait statefulset.apps/pachd-loki --for=jsonpath='{.status.readyReplicas}'=1 --timeout=5m
```

Comment:
Can we get rid of this eventually? Should we track this in a ticket?

Reply:
I will make a ticket. I checked to see if the issue was resolved when I made the pr and it is still not fixed :(